proto/rill/local/v1/api.proto

```diff
@@ -42,3 +51,56 @@ message GetVersionResponse {
   string current = 1;
   string latest = 2;
 }
+
+message DeployValidationRequest {}
+
+message DeployValidationResponse {
+  bool is_authenticated = 1; // if true below fields are relevant after login
+  bool is_github_connected = 2; // if true below fields are relevant after git install
+  string git_grant_access_url = 3; // redirect to this if is_github_connected or is_repo_access_granted is false
+  string git_user_name = 4;
+  bool is_user_app_installed = 5; // if rill git app is installed on user account
+  string user_app_permission = 6;
+  repeated string git_user_orgs = 7;
```

Comment:
1. How is this different from `organizations_with_app`?
2. Also, "Github" not "Git" :)

Reply:
This list all the users orgs where as `organizations_with_app` lists only the orgs on which Rill github app is installed. not sure what would be the use of `git_user_orgs`, may be we can merge both the lists and orgs on which app is not installed we set the permission as empty string.